build/yamls/antrea.yml

```diff
@@ -1047,6 +1047,9 @@ data:
     # Enable traceflow which provides packet tracing feature to diagnose network issue.
     #  Traceflow: false
 
+    # Enable NodePortLocal feature to make the pod reachable externally through nodeport
```

Comment:
You should change build/yamls/base/conf/antrea-controller.conf, and do "make manifest" which will update all yamls.